src/main/java/org/jenkinsci/plugins/workflow/graphanalysis/FlowScanningUtils.java

```diff
@@ -59,6 +62,53 @@ public boolean apply(FlowNode input) {
     // Default predicates, which may be used for common conditions
     public static final Predicate<FlowNode> MATCH_BLOCK_START = (Predicate)Predicates.instanceOf(BlockStartNode.class);
 
+    /** Sorts flownodes putting the one begun last (oldest startTime) at the end, with null times last
+     *  because likely they represent the newest nodes with a {@link TimingAction} not attached yet. */
```

Comment:
This now seems unused, either in this PR or the downstreams I can see. Added deliberately?